cms/envs/common.py

```diff
@@ -551,6 +551,17 @@
     #   https://github.com/openedx/openedx-events/issues/265
     # .. toggle_tickets: https://github.com/edx/edx-arch-experiments/issues/381
     'ENABLE_SEND_XBLOCK_LIFECYCLE_EVENTS_OVER_BUS': False,
+
+    # .. toggle_name: FEATURES['ENABLE_HIDE_FROM_TOC_UI']
```

Comment:
Note: There is an open issue that claims an intent to deprecate the features dict in favor of top-level settings. Each time I see a new one, I’m reminded that we haven’t actually made a decision around that. This PR doesn’t necessarily need to change. FYI: @kdmccormick

Reply:
Thank you for the heads up. In the meantime, should we avoid adding new feature entries? If so, could add a message somewhere with this suggestion?

Reply:
@mariajgrimaldi: See https://github.com/openedx/edx-platform/issues/33026 for more details. It also discusses a potential change in naming convention as well. You are welcome to participate in that discussion. And, I do **not** think this PR needs to change at this time. Hopefully someday we'll have more clarity on this front.

Reply:
Thank you!